PrReviewCode/Program.cs

```diff
@@ -4,6 +4,8 @@ public class Program
 {
     public static void Main(string[] args)
     {
+        throw new Exception("Test Exception");
```

Comment:
Unconditional exception throw here will crash the program. Remove or add proper error handling logic.

 **SUGGESTION:**

```suggestion
 // Remove this line or add a conditional check before throwing
```